### PR TITLE
Support multiple stylesheets types

### PR DESCRIPTION
**Pull request by cesarizu**
This is a PR that implements sapegin/grunt-webfont#190

### CODE REVIEW

**Comment by sapegin** [comment]
Please fix the build first.

**Comment by cesarizu** [comment]
It's fixed now and I added a test for the multiple stylesheets options.

File: tasks/webfont.js [hunk 1]
```diff
@@ -310,7 +325,28 @@ module.exports = function(grunt) {
...
+				codepoints.push(o.codepoints[name].toString(16));
+			});
+			o.codepoints = codepoints;
+
+			// Prepage glyph names to use as CSS classes
+			o.glyphs = _.map(o.glyphs, classnameize);
+
+			o.stylesheets.forEach(generateStylesheet);
+
+			done();
```
Comment (sapegin) [inline tasks/webfont.js]: You’re calling `done` as soon as you start generation, not when it’s finished.

File: tasks/webfont.js [hunk 2]
```diff
@@ -310,7 +325,28 @@ module.exports = function(grunt) {
...
+		}
+
+		/**
+		 * Generate CSS
+		 *
+		 * @param {Stylesheet} stylesheet type: css, scss, ...
```
Comment (sapegin) [inline tasks/webfont.js]: Stylesheet → string?

File: tasks/webfont.js (hunk 1 above)
Comment (cesarizu) [inline tasks/webfont.js]: I'm not really sure what you mean here. `done()` is called after calling `generateStylesheet` for every type of stylesheet needed.

File: tasks/webfont.js (hunk 2 above)
Comment (cesarizu) [inline tasks/webfont.js]: Thanks for spotting that :) fixed....

File: tasks/webfont.js (hunk 1 above)
Comment (sapegin) [inline tasks/webfont.js]: If `generateStylesheet` synchronous than it’s fine.

Comment (cesarizu) [inline tasks/webfont.js]: Yes, all the code is synchronous and it ends with a `fs.writeFileSync` so it waits for the file to be written before continuing (same as before this PR).

**Pull request merged**